src/test/coll.rs

```diff
@@ -541,7 +541,7 @@ async fn empty_insert() {
         .database(function_name!())
         .collection::<Document>(function_name!());
     match coll
-        .insert_many(Vec::new(), None)
+        .insert_many(Vec::<Document>::new(), None)
```

Comment:
In certain cases, this will make type inference more difficult and require the usage of type arguments, such as in this case. I think this is worth it over taking in just `&T` though, which rules out owned values and other smart-pointer types from being passed in (e.g. `Arc`, `Box`). I think in most cases users won't be relying on the concrete type in these methods and the annotations won't be necessary.